nuxt-opengraph/components/OgImage/collection.vue

```diff
@@ -0,0 +1,62 @@
+<script setup lang="ts">
+import type { CSSProperties } from 'vue';
+
+// inherited attrs can mess up the satori parser
+defineOptions({
+  inheritAttrs: false,
+});
+
+defineProps<{
+  title: string;
+  image: string;
+  items: string;
+  // owners: string
+  network: string;
+}>();
+
+const cover: ComputedRef<CSSProperties> = computed(() => {
+  return {
+    objectFit: 'cover',
+    objectPosition: 'center',
+  };
+});
+</script>
+
+<template>
+  <img :src="image" :alt="title" :style="cover" class="h-full w-full" />
+
+  <div
+    class="flex flex-col justify-end h-full w-full bg-slate-900/85 text-white p-10 text-2xl font-bold absolute inset-0"
```

Comment:
**question**: how is tailwind imported here?

Reply:
already builtin from satori itself https://nuxtseo.com/og-image/guides/using-satori